src/aggregations/aggregation-builder.js

```diff
@@ -0,0 +1,64 @@
+import _ from 'lodash'
+import aggregations from './index'
+
+const AggregationBuilder = {
```

Comment:
Many reasons. First one being that `Object.create` is _not_ an alternative to `new`. If anything, you could achieve the same using something like:

```javascript
function X() {}
var o = Object.create(X.prototype) // This is essentially new X()
```
Javascript has no classes, despite whatever the literature or SO questions may love to express. The `class` syntax is (almost) just syntactic sugar on top of the classic `[[Prototype]]` mechanism. 

And here, we don't need all that sugar. We just need to _compose_ objects. Aggregations modules need to "become builders". The alternative would be to create a class and have all your `*-aggregation.js` functions become classes that extended the builder, thus modifying 30% of the files in the project.

I'd _highly_ recommend reading YDKJS for more on composition and javascript class illness: https://github.com/getify/You-Dont-Know-JS/blob/master/this%20&%20object%20prototypes/ch6.md